### ISSUE TITLE
The generator loss is so high 

### ISSUE THREAD

**Issue by Ng-ms** [thomasneff/Generative-Adversarial-Network-based-Synthesis-for-Supervised-Medical-Image-Segmentation#10]
The code is running for my data now but the generator loss is up to 10 while the d loss is 0.0004 
What should I change to make it work?

**Comment by thomasneff** [comment]
Unfortunately, I don't remember the "typical ranges" of the generator and discriminator loss anymore - it's been quite a while. There are a million different things that could be going wrong here, which I can not really help you to debug. 

You could try: Looking at intermediate network outputs and making sure that the input data is processed correctly and is in the correct range. Otherwise, if you're using the same dataset as is used in the paper, you should be able to get some reasonable results from this. 

If you're trying to use a different dataset, then you might run into much more difficult problems and might potentially need to fine-tune the architecture, batch sizes, training epochs, weight initialization,  and much more...

**Comment by Ng-ms** [comment]
I am using it on another Drive dataset , 
Also should I do the traditional augmentation before the Gan  or not
